docs/source/notebooks/subset-user-input.ipynb

```diff
@@ -1272,7 +1338,7 @@
     {
      "data": {
       "text/plain": [
-       "'https://pavics.ouranos.ca/wpsoutputs/0dfbc373-ce29-11eb-95d5-8962d32ec48d/nrcan_v2_sub.ncml'"
+       "'https://host-140-91.rdext.crim.ca/wpsoutputs/0677460a-1b1e-11ec-8812-55fea68da9e0/nrcan_v2_sub.ncml'"
```

Comment:
We have to find replace all `host-140-91.rdext.crim.ca` to `pavics.ouranos.ca` ... it's in the output as well